File: app/src/main/java/com/example/proyecto/Util/New/QuoteRepository.java
package com.example.proyecto.Util.New;

import com.example.proyecto.Entidades.Unit;

import java.util.ArrayList;
import java.util.List;

public class QuoteRepository {

    private QuoteService quoteService = new QuoteService();

    public List<Unit> getAllQuotes(){

        List<Unit> response = quoteService.getQuotes();
        QuoteProvider.quotes = response;

        return response;
    }
}

File: app/src/main/java/com/example/proyecto/Util/New/Kotlinclass/QuoteRepository.kt
package com.example.proyecto.Util.New.Kotlinclass

class QuoteRepository {
    private val api = QuoteService()

    suspend fun getAllQuotes(codigo:String):List<QuoteModelUnidadSesion>{
        val response:List<QuoteModelUnidadSesion> = api.getQuotes(codigo)
        QuoteProvider.quotes = response
        return response
    }
}